### PR TITLE
Update Swedish tech stack

### PR DESCRIPTION
**Pull request by nanjiangshu**
This PR update the tech stack for the Swedish node.

## Types of changes

- [x] Modified content (non-breaking change which modifies existing content)

## Checklist:

- [ ] I have read the CONTRIBUTING.md guide.
- [x] My changes follow the code style of this project.
- [ ] I have updated supporting documentation, if required.
- [ ] I have added my details to the CONTRIBUTORS.yaml file.

### CODE REVIEW

File: docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html [hunk 1]
```diff
@@ -45,17 +45,17 @@
...
       <td class="tg-center-bold">9</td>
       <td class="tg-left-bold">Backup Storage</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">S3</td>
     </tr>
     <tr>
       <td class="tg-center-bold">10</td>
       <td class="tg-left-bold">Main Source Code Repository</td>
-      <td class="tg-left"><a href="https://github.com/neicnordic/sda-pipeline">https://github.com/neicnordic/sda-pipeline</a></td>
+      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/tree/main/sda-pipeline/">https://github.com/neicnordic/sensitive-data-archive/tree/main/sda-pipeline</a></td>
```
Comment (jbygdell) [inline docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html:58]:
```suggestion
      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/">https://github.com/neicnordic/sensitive-data-archive/</a></td>
```

File: docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html [hunk 2]
```diff
@@ -10,32 +10,32 @@
...
       <td class="tg-center-bold">4</td>
       <td class="tg-left-bold">FEGA Node MQ</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">Implemented</td>
     </tr>
     <tr>
       <td class="tg-center-bold">5</td>
       <td class="tg-left-bold">Submitter Inbox</td>
-      <td class="tg-left">S3 Proxy</td>
+      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/tree/main/sda/cmd/s3inbox">S3 Proxy</a></td>
```
Comment (jbygdell) [inline docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html:33]:
```suggestion
      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/">S3 Proxy</a></td>
```

File: docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html [hunk 3]
```diff
@@ -10,32 +10,32 @@
...
       <td class="tg-center-bold">2</td>
       <td class="tg-left-bold">Credentials</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">Implemented</td>
     </tr>
     <tr>
       <td class="tg-center-bold">3</td>
       <td class="tg-left-bold">FEGA Node Encryption key pair</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">Implemented</Img></td>
```
Comment (nanjiangshu) [inline docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html:23]:
```suggestion
      <td class="tg-left">Generated with <a href="https://github.com/neicnordic/crypt4gh/">Crypt4GH</a> tool</td>
```

File: docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html [hunk 4]
```diff
@@ -10,32 +10,32 @@
...
       <td class="tg-center-bold">1</td>
       <td class="tg-left-bold">URL</td>
-      <td class="tg-left"><a href="https://nbis.se/infrastructure/sensitive-data-archive.html">https://nbis.se/infrastructure/sensitive-data-archive.html</a></td>
+      <td class="tg-left"><a href="https://fega.nbis.se/">https://fega.nbis.se/</a></td>
     </tr>
     <tr>
       <td class="tg-center-bold">2</td>
       <td class="tg-left-bold">Credentials</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">Implemented</td>
```
Comment (nanjiangshu) [inline docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html:18]:
```suggestion
      <td class="tg-left">User: LS Login and CEGA username and password, Services: credentials to connect to CEGA NSS and RMQ endpoints</td>
```

**Comment by malloryfreeberg** [comment]
@jbygdell @nanjiangshu - feel free to add me as a reviewer when you're ready for the PR to be reviewed :) And please do add yourselves to the [CONTRIBUTORS.yaml](https://github.com/EGA-archive/FEGA-onboarding/blob/main/CONTRIBUTORS.yaml) file for attribution.

File: docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html [hunk 5]
```diff
@@ -10,32 +10,32 @@
...
       <td class="tg-center-bold">4</td>
       <td class="tg-left-bold">FEGA Node MQ</td>
-      <td class="tg-left"></td>
+      <td class="tg-left">Rabbit MQ</td>
     </tr>
     <tr>
       <td class="tg-center-bold">5</td>
       <td class="tg-left-bold">Submitter Inbox</td>
-      <td class="tg-left">S3 Proxy</td>
+      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/">S3 Proxy</a></td>
```
Comment (jbygdell) [inline docs/topics/technical-operational/countries_tech_stack/sweden_tech_stack.html:33]:
```suggestion
      <td class="tg-left"><a href="https://github.com/neicnordic/sensitive-data-archive/">S3Inbox</a></td>
```

**Review by pahatz — APPROVED** [review]
Looks good!

**Comment by nanjiangshu** [comment]
Hi @malloryfreeberg, thanks a lot for your comment. I tried to add you as a reviewer but it seems I don't have permission for this repo. I want to let you know that this PR is ready and one of my colleagues has approved it already.

**Comment by malloryfreeberg** [comment]
Hi @nanjiangshu - thank you for the ping. Please confirm whether you want to add yourself to the [CONTRIBUTORS.yaml](https://github.com/EGA-archive/FEGA-onboarding/blob/main/CONTRIBUTORS.yaml) file for attribution. If so, please make the change and then I will approve/merge. Thanks!

**Comment by nanjiangshu** [comment]
Thanks @malloryfreeberg. I've created a PR to add my name to the contributors list.

**Review by malloryfreeberg — APPROVED** [review]

**Pull request merged**